api/src/main/java/io/lumeer/engine/api/constraint/CaseConstraintType.java

```diff
@@ -47,6 +52,30 @@
       return Collections.singleton(CONSTRAINT_PREFIX);
    }
 
+   static BiFunction<Object, Class, Object> getEncodeFunction() {
+      return (o, t) -> {
+         if (t != null && t != String.class) {
+            return null;
```

Comment:
We accept only String.class request. So either the type is not specified or String is requested.